requirements.txt

```diff
@@ -39,24 +39,24 @@ pycparser==2.20           # via cffi
 pydot2==1.0.33            # via reana-workflow-engine-yadage (setup.py)
 pydotplus==2.0.2          # via reana-workflow-engine-yadage (setup.py)
 pygraphviz==1.7           # via reana-workflow-engine-yadage (setup.py)
-pyopenssl==19.0.0         # via requests
+pyopenssl==20.0.1         # via requests
 pyparsing==2.4.7          # via pydot2, pydotplus
 pyrsistent==0.17.3        # via jsonschema
 python-dateutil==2.8.1    # via bravado, bravado-core
 pytz==2021.1              # via bravado-core, fs
 pyyaml==5.4.1             # via bravado, bravado-core, packtivity, reana-commons, swagger-spec-validator, yadage, yadage-schemas
 reana-commons[yadage]==0.8.0a17  # via reana-workflow-engine-yadage (setup.py)
-requests[security]==2.22.0  # via bravado, packtivity, reana-workflow-engine-yadage (setup.py), yadage, yadage-schemas
+requests[security]==2.25.1  # via bravado, packtivity, reana-workflow-engine-yadage (setup.py), yadage, yadage-schemas
 rfc3987==1.3.8            # via jsonschema, reana-workflow-engine-yadage (setup.py)
 simplejson==3.17.2        # via bravado, bravado-core
-six==1.15.0               # via bravado, bravado-core, fs, jsonpath-rw, jsonschema, mock, pyopenssl, python-dateutil, swagger-spec-validator, webcolors
+six==1.16.0               # via bravado, bravado-core, fs, jsonpath-rw, jsonschema, mock, pyopenssl, python-dateutil, swagger-spec-validator
 strict-rfc3339==0.7       # via jsonschema
 swagger-spec-validator==2.7.3  # via bravado-core
 typing-extensions==3.10.0.0  # via bravado
-urllib3==1.25.11          # via requests
+urllib3==1.26.5           # via requests
 vine==1.3.0               # via amqp
-webcolors==1.9.1          # via jsonschema
-werkzeug==1.0.1           # via reana-commons
+webcolors==1.11.1         # via jsonschema
+werkzeug==2.0.1           # via reana-commons
```

Comment:
Werkzeug 2 works well here, but I wonder whether we don't use the same `werkzeug < 2` pin in `r-w-e-*` components as in the other cluster modules for overall homogeneity? We can go either way since I guess we don't use any deep werkzeug features _directly_ ourselves besides local proxy... Most of the usage is indirect.

Reply:
In case we wanted to upper pin `werkzeug < 2` I think this should come [from `r-commons`](https://github.com/reanahub/reana-commons/blob/347ac3670bd34b9cd004c4ef4a747db15da05876/setup.py#L52).

But as you said, and don't think it'd affect much.